ftlib/commlib/gloo/src/gloo_lib.cxx

```diff
@@ -0,0 +1,99 @@
+#include <pybind11/pybind11.h>
+#include <pybind11/numpy.h>
+#include <pybind11/stl.h>
+
+#include <iostream>
+#include <vector>
+#include <string>
+#include <chrono>
+
+#include <assert.h>
+#include <stdlib.h>
+
+#include "gloo/transport/tcp/device.h"
```

Comment:
Where did Gloo get installed? e.g. if users want to use this, how should they install FTLib to be able to use Gloo implementation?

Reply:
I'll add a dockerfile to this pr.